sap/5.2/modules/ROOT/pages/sap-connector-reference.adoc

```diff
@@ -0,0 +1,638 @@
+| Target Variable a| String |  Name of the variable in which to store the operation's output |  |
+| Target Value a| String |  Expression that evaluates the operation's output. The expression outcome is stored in the target variable. |  #[payload] |
+| Reconnection Strategy a| * <<reconnect>>
+* <<reconnect-forever>> |  A retry strategy in case of connectivity errors. |  |
+|===
+
+==== Output
+[%autowidth.spread]
+|===
+|Type |Binary
+|===
+
+=== For Configurations
+* <<sap>>
+
+==== Throws
+* SAP:CONNECTIVITY
+* SAP:INVALID_CACHE
+* SAP:INVALID_INPUT
+* SAP:METADATA_UNAVAILABLE
+* SAP:NOT_FOUND
+* SAP:NOT_SUPPORTED
+* SAP:PARSING
+* SAP:RETRY_EXHAUSTED
+* SAP:SYSTEM_BUSY
+* SAP:TIMEOUT
+
+== Types
+[[MessageServer]]
+=== Message Server
+
+[%header,cols="20s,25a,30a,15a,10a"]
+|===
+| Field | Type | Description | Default Value | Required
+| Host a| String | The host of the message server. |  | x
+| System Id a| String | System ID of the SAP system. |  | x
+| Port a| Number | The port with which the connector will log into the message server. |  |
+| Group a| String | Group of SAP application servers. |  |
+| Router a| String | SAP router string to use for a system protected by a firewall. |  |
+|===
+
+[[Reconnection]]
+=== Reconnection
+
+[%header,cols="20s,25a,30a,15a,10a"]
+|===
+| Field | Type | Description | Default Value | Required
+| Fails Deployment a| Boolean | When the application is deployed, a connectivity test is performed on all connectors. If set to `true`, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy. |  |
+| Reconnection Strategy a| * <<reconnect>>
+* <<reconnect-forever>> | The reconnection strategy to use. |  |
+|===
+
+[[reconnect]]
+=== Reconnect
+
+[%header,cols="20s,25a,30a,15a,10a"]
+|===
+| Field | Type | Description | Default Value | Required
+| Frequency a| Number | How often to reconnect (in milliseconds). | |
+| Count a| Number | The number of reconnection attempts to make. | |
+|===
+
+[[reconnect-forever]]
+=== Reconnect Forever
+
+[%header,cols="20s,25a,30a,15a,10a"]
+|===
+| Field | Type | Description | Default Value | Required
+| Frequency a| Number | How often in milliseconds to reconnect. | |
+|===
+
+[[ExpirationPolicy]]
+=== Expiration Policy
+
+[%header,cols="20s,25a,30a,15a,10a"]
+|===
+| Field | Type | Description | Default Value | Required
+| Max Idle Time a| Number | A scalar time value for the maximum amount of time a dynamic configuration instance should be allowed to be idle before it's considered eligible for expiration. |  |
+| Time Unit a| Enumeration, one of:
+
+** NANOSECONDS
+** MICROSECONDS
+** MILLISECONDS
+** SECONDS
+** MINUTES
+** HOURS
+** DAYS | A time unit that qualifies the maxIdleTime attribute. |  |
+|===
+
+[[SapAttributes]]
+=== SAP Attributes
+
+[%header,cols="20s,25a,30a,15a,10a"]
+|===
+| Field | Type | Description | Default Value | Required
+| Transaction Id a| String |Transaction ID value.  |  |
+|===
+
```

Comment:
please add this:

[[IDocRenderParameterGroup]]
=== I Doc Render Parameter Group

[cols=".^20%,.^25%,.^30%,.^15%,.^10%", options="header"]
|======================
| Field | Type | Description | Default Value | Required
| Render I Doc Empty Tags a| Boolean | Rendering option for including also all empty tags in the resulting XML document. | false | 
| Render I Doc Ignore Invalid Char Errors a| Boolean | Rendering option for ignoring character range check errors in accordance with the used XML specification version. | false | 
| Render I Doc Ignore Tag Escape Errors a| Boolean | Rendering option for ignoring escape character errors within XML tags in accordance with the used IDocXML format. | false | 
| Render I Doc Omit Xml Declaration a| Boolean | Rendering option for omitting the XML declaration section in the resulting XML document. | false | 
| Render I Doc Optional Encoding Decl a| Boolean | Rendering option for writing also an optional XML encoding declaration attribute into the resulting XML document. | false | 
| Render I Doc Substitute Invalid Chars a| Boolean | Rendering option for substituting invalid characters within IDoc field values with the number sign '#' (U+0023). | false | 
| Render I Doc With CR a| Boolean | Rendering option for inserting CR (Carriage Return) characters into the resulting XML document. | false | 
| Render I Doc With CRLF a| Boolean | Rendering option for inserting CR+LF (Carriage Return and Line Feed) characters into the resulting XML document. | false | 
| Render I Doc With LF a| Boolean | Rendering option for inserting LF (Line Feed) characters into the resulting XML document. | false | 
| Render I Doc With Tabs And CR a| Boolean | Rendering option for inserting TAB and CR (Tabulators and Carriage Return) characters into the resulting XML document. | false | 
| Render I Doc With Tabs And CRLF a| Boolean | Rendering option for inserting TAB and CR+LF (Tabulators, Carriage Return and Line Feed) characters into the resulting XML document. | false | 
| Render I Doc With Tabs And LF a| Boolean | Rendering option for inserting TAB and LF (Tabulators and Line Feed) characters into the resulting XML document. | false | 
| Render I Doc Xml10 a| Boolean | Rendering option for creating an XML document compliant to the XML specification version 1.0 | false | 
| Render I Doc Xml11 a| Boolean | Rendering option for creating an XML document compliant to the XML specification version 1.1 | false | 
|======================